src/Workspaces/Core/Portable/Workspace/Solution/ProjectDependencyGraph.cs

```diff
@@ -439,6 +724,21 @@ private void ComputeTransitiveReferences(ProjectId project, HashSet<ProjectId> r
             }
         }
 
+        /// <summary>
+        /// Gets the list of projects that directly or transitively depend on this project, if it has already been
+        /// cached.
+        /// </summary>
+        internal ImmutableHashSet<ProjectId>? TryGetProjectsThatTransitivelyDependOnThisProject(ProjectId projectId)
```

Comment:
Mark as test only if it's only being used for tests? I worry about anybody taking a dependency on this.

Reply:
Moved to test accessor locally; will mark resolved on push

Reply:
➡️ Now pushed.